docs/deployment/uwsgi_cookiecutter_nginx.rst

```diff
@@ -0,0 +1,233 @@
+.. _uwsgi_tutorial:
+
+Running a Cookiecutter :app:`Pyramid` Application under :app:`uWSGI` and :app:`Nginx`
+=====================================================================================
+
+``uWSGI`` is a software application for building hosting services.
+It is named after the Web Server Gateway Interface (the WSGI specification
+to which many Python web frameworks conform).
+
+This guide will outline broad steps that can be used to get a Cookiecutter
+:app:`Pyramid` application running under Nginx via ``uWSGI``.  This particular
+tutorial was developed under Ubuntu 18.04, but the instructions should be
+largely the same for all systems, delta specific path information for commands
+and files.
+
+
+#.  Install prerequisites
+
+    .. code-block:: bash
+
+       $ sudo apt install -y uwsgi-core uwsgi-plugin-python3 python3-cookiecutter \
+                             python3-pip python3-venv nginx
+       $ python3 -m cookiecutter gh:Pylons/pyramid-cookiecutter-starter --checkout master
+
+#.  Create a :app:`Pyramid` application. For this tutorial we'll use the
+    ``starter`` :term:`cookiecutter`. See :ref:`project_narr` for more
+    in-depth information about creating a new project.
+
+    .. code-block:: bash
+
+       $ cd
+       $ cookiecutter gh:Pylons/pyramid-cookiecutter-starter --checkout master
```

Comment:
See above comment, and possibly update for consistency.

Also please use 4 spaces for indents for consistency.